### ISSUE TITLE
[bug] [minor] Inconsistent slash direction in xml file loading paths

### ISSUE THREAD

**Issue by izik1** [JapaMala/armok-vision#40]
Steps to reproduce: 
1. Open armok vision
2. While it is loading the xml files look at the slashes
3. The last two slashes are backslashes whereas all the other ones are forward
Result:
![armok vision bug](https://cloud.githubusercontent.com/assets/21113608/22614635/ad54de40-ea39-11e6-8ae7-a86d03e46b22.png) (user directory is the empty one. actually user directory name hidden for privacy purposes)

**Comment by lethosor** [comment]
This should work on Windows, but if the backslashes are hardcoded, it could cause issues like #2 on Linux/OS X.

**Comment by izik1** [comment]
perhaps as a fix the os can be detected using [Application.Platform](https://docs.unity3d.com/ScriptReference/Application-platform.html) and insert forward / backslashes based on that?

**Comment by lethosor** [comment]
Forward slashes should work on Windows.

**Comment by izik1** [comment]
Oh, I forgot a major part of any bug report. I am running windows 7. So it probably really is just a cosmetic issue (the reason why I originally suggested minor as a label) the slash direction is probably default for the OS or unity. Not sure though.

**Comment by lethosor** [comment]
Yeah, the "U:/" in the path indicated that it was Windows. I'm not sure if Armok Vision or Unity is choosing the type of slash here. If Armok Vision is always choosing backslashes, it would break on *other* platforms.

(There wasn't a "minor" label to choose from before, but I'll add it.)
